docs/tutorial-smart-ai-1.md

```diff
@@ -0,0 +1,208 @@
+# Introduction to SmartAI
+
+## What type of waypoint will I use?
+
+[WIP]
+
+SmartAI allows us to use 2 different waypoint tables:
+
+waypoint_data, which is used in cpp and in more general patrols
+waypoints, a type which was made specifically for SmartAI
+
+[WIP]
+
+# Scripting a Simple Scripted Patrol
```

Comment:
All headers under the "first header" should be 2 # for a "Main header" and more for underheaders.

```
# Top Header - the only one which is allowed to use one number sign

## Main Header 1

### Under Header 1-1

### Under Header 1-2

## Main Header 2

## Main Header 3